### ISSUE TITLE
[Request] Add the ability to play sound sequences (bfseq)

### ISSUE THREAD

**Issue by JustaRandomUser12** [Gota7/Citric-Composer#30]
I want to work with bfseq's for the Wii U Menu but I can't do it. Curious if a feature like this could be possible.

**Comment by RedStoneMatt** [comment]
Agreed. This tool seems to unfotunately be dead, it's quite a shame!
Playing sequenced musics like Nitro Studio does would be huge to rip musics from games that don't stream them
